README.md

```diff
@@ -1,16 +1,205 @@
+
+```js
+function myFunc () {
+  const myVar = 42;
+
+  return myVar * 2;
+}
+// => undefined
+
+myFunc();
+// => 84
+```
+
+However, from outside the function, we can't reference anything declared inside
+of it:
+
+```js
+function myFunc () {
+  const myVar = 42;
+}
+// => undefined
+
+myVar * 2;
+// Uncaught ReferenceError: myVar is not defined
+```
+
+The function body creates its own scope. It's like a separate channel on Slack —
+only the members of that channel can read the messages sent in it.
+
+### Block Scope
+
+A block statement also creates its own scope... kind of. As of ES2015,
+JavaScript has partial support for block scoping.
+
+Variables declared with `var` are **not** block-scoped:
+
+```js
+if (true) {
+  var myVar = 42;
+}
+
+myVar;
+// => 42
+```
+
+However, variables declared with `const` and `let` **are** block-scoped:
+
+```js
+if (true) {
+  const myVar = 42;
+
+  let myOtherVar = 9001;
+}
+
+myVar;
+// Uncaught ReferenceError: myVar is not defined
+
+myOtherVar;
+// Uncaught ReferenceError: myOtherVar is not defined
+```
+
+This is yet another reason to **never use `var`**. As long as you stick to
+declaring variables with `const` and `let`, what happens in block stays in
+block.
+
+## Identify Best Practices in Defining Scope
+
+In a perfect world, you'd always remember to declare new variables with `const`
+and `let`, and you'd never run into any unfortunate scoping issues. However,
+it's inevitable that at some point you're going to forget the `const` or `let`
+and accidentally do something like:
+
+```js
+firstName = 'Ada';
+```
+
+Variables created without a `const`, `let`, or `var` keyword are **always
+globally-scoped**, regardless of where they sit in your code. If you create one
+inside of a block, it's still available globally:
+
+```js
+if (true) {
+  lastName = 'Lovelace';
+}
+
+lastName;
+// => "Lovelace"
+```
+
+If you create one inside of a function, it's still available globally:
+
+```js
+function bankAccount () {
+  secretPassword = 'il0v3pupp135';
+
+  return 'bankAccount() function invoked!';
+}
+
+bankAccount();
+// => "bankAccount() function invoked!"
+
+secretPassword;
+// => "il0v3pupp135"
+```
+
+Oh no; our super secret password has leaked into the global scope and is
+available everywhere! Declaring global variables and functions should only be
+used as a last resort if you absolutely need access to something **everywhere**
+in your program. In general, it's best practice to make variables and functions
+available only where they're needed — and nowhere else. Making a variable
+available in places that shouldn't have access to it can only lead to bad things
+and make your debugging process more complex. The more pieces of code that can
+access a given variable, the more places you have to check for bugs if/when that
+variable contains an unexpected value. So keep your scopes small and specific.
 
 ## Conclusion
 
+We figured out how scope works in JavaScript. We know that our declared
```

Comment:
I wouldn't use language that assumes students all know or understand what's been discussed. We can mention: "we've talked about"... "We detailed how"... "we've reviewed/discussed"... Or even "We've seen that"... etc.